etl/steps/data/garden/cedlas/2024-03-08/sedlac.meta.yml

```diff
@@ -0,0 +1,749 @@
+
+      In those surveys where this information is not available or is clearly unreliable, we increase household income of housing owners by 10%, a value that is consistent with estimates of implicit rents in the region.
+
+    presentation:
+      topic_tags:
+        - Poverty
+        - Economic Inequality
+
+  poverty_lines: |-
+    <% if table == "$2.15 a day" %>
+    Extreme poverty here is defined as living below the International Poverty Line of $2.15 per day.
+    <% elif table == "$3.65 a day" %>
+    A poverty line of $3.65 a day represents definitions of national poverty lines in lower-middle-income countries.
+    <% elif table == "$6.85 a day" %>
+    A poverty line of $6.85 a day represents definitions of national poverty lines in upper-middle-income countries.
+    <% elif table == "50% of median" %>
+    This is a measure of _relative_ poverty – it captures the share of people whose income is low by the standards typical in their own country.
+    <%- endif -%>
+
+  ppps: |-
+    The data is measured in international-$ at 2017 prices – this adjusts for inflation and for differences in the cost of living between countries.
+
+  welfare_equivalence: |-
+    <% if table == "$2.15 a day" or table == "$3.65 a day" or table == "$6.85 a day" or table == "50% of median" or table == "Per capita income"%>
+    This is household income per capita. 'Per capita' means that the income of each household is attributed equally to each member of the household (including children).
+    <% elif table == "Equivalized income" %>
+    Household income has been equivalized – adjusted to account for the fact that people in the same household can share costs like rent and heating.
+    <%- endif -%>
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 365
+
+
+tables:
+  sedlac:
+    variables:
+      national_headcount_ratio:
+        title: <<table>> - Share of population in poverty (Spell <<survey_number>>)
+        description_short: Percentage of population living in households with an income or consumption per person below <<table.lower()>>
+        description_key:
+          - "{definitions.poverty_lines}"
+          - "{definitions.ppps}"
+          - "{definitions.welfare_equivalence}"
+        unit: '%'
+        short_unit: '%'
+        display:
+          name: <<table>> - Share of population in poverty
+          <<: *common-display
+        presentation:
+          title_public: <<table>> - Share of population in poverty
+
+      national_poverty_gap_index:
+        title: <<table>> - Poverty gap index (Spell <<survey_number>>)
+        description_short: The poverty gap index is a poverty measure that reflects both the prevalence and the depth of poverty. It is calculated as the share of population in poverty multiplied by the average shortfall from the poverty line (expressed as a % of the poverty line).
```

Comment:
can just be - "The poverty gap index reflects both the prevalence and the depth of poverty. "

Reply:
We have it standardized as that in our guides, so I will keep it for now.